concourse/pipeline.yml

```diff
@@ -28,7 +28,24 @@ resources:
       branch: master
       pro: true
 
+  - name: feature-tests-image
+    type: docker-image
+    icon: docker
+    source:
+      repository: ((readonly_private_ecr_repo_url))
+      tag: govuk-coronavirus-vulnerable-people-form-feature-tests
+
 jobs:
+  - name: build-feature-tests-image
+    serial: true
+    plan:
+      - get: govuk-coronavirus-vulnerable-people-form
+        trigger: true
```

Comment:
Building this on every commit feels excessive. It's not going to change that much. Can we put it on a schedule instead?

Reply:
We could have a second git resource for just the bits of the repo that might change the image (I think `Gemfile*`, `.ruby-version` and `features/*`), or yeah, we could just do it daily.

Reply:
Had a stab at that in f50d9a3 - untested though. Could either merge this and deal with any mistakes afterwards or point the pipeline at this branch (with prod paused). Whaddyathink?

Reply:
Merge and deal with mistakes afterwards is easier I reckon.